src/cli/self_update.rs

```diff
@@ -58,14 +58,16 @@ mod shell;
 mod unix;
 #[cfg(windows)]
 mod windows;
+#[cfg(windows)]
+pub use windows::complete_windows_uninstall;
```

Comment:
This probably shouldn't be interleaved with the mod definitions; It should just sit in the same-crate imports; I think I tweaked this recently, but if it isn't conflicting, don't worry about it.